src/types/UiaTextRangeBase.cpp

```diff
@@ -405,16 +405,22 @@ std::optional<bool> UiaTextRangeBase::_verifyAttr(TEXTATTRIBUTEID attributeId, V
         THROW_HR_IF(E_INVALIDARG, val.vt != VT_I4);
 
         // The underline style is stored as a TextDecorationLineStyle.
-        // However, The text buffer doesn't have that many different styles for being underlined.
-        // Instead, we only have single and double underlined.
+        // However, The text buffer doesn't have all the different styles for being underlined.
+        // Instead, we only use a subset of them.
         switch (val.lVal)

```

Comment:
I'm still not clear how UIA stuff works currently, so I may need someone to crosscheck whether the changes work as expected. I simply did what made sense, without **any** testing.

Reply:
This is fine -- thanks!

Reply:
Thanks for checking!